tests/functional/pages.php

```diff
@@ -0,0 +1,63 @@
+<?php
+define('INSTALL_ROOT',  realpath(__DIR__ . '/../../') . '/');
+
+// We always work with UTF8 encoding
+mb_internal_encoding('UTF-8');
+
+// Make sure we have a timezone set
+date_default_timezone_set('Europe/Paris');
+
+require __DIR__ . '/../../vendor/autoload.php';
+
+// Set an environment variable so that the instance will use content from test files
+putenv("AUTOMATED_TESTS=true");
+
+// Launch PHP dev server in the background
+chdir(INSTALL_ROOT);
+exec('./start.sh -remote > /dev/null 2>&1 & echo $!', $output);
+
+// We will need the pid to kill it, beware, this is the pid of the bash process started with start.sh
+$pid = $output[0];
+
+// Pause to let time for the dev server to launch in the background
+sleep(3);
+
+$paths = [
+    ['channelcomparizon/', 200, 'Compare strings from channel to channel', 'Key'],
+    ['channelcomparison/', 200, 'Compare strings from channel to channel', 'Key'],
+    ['credits/', 200, 'Transvision 1.0 was created', 'Transvision is hosted on the MozFR server'],
+    ['downloads/', 200, 'Select which strings', 'Generate the TMX'],
+    ['gaia/', 200, 'Translation Status', 'How many strings are translated'],
+    ['news/', 200, 'Version 4.0', 'End user visible changes'],
+    ['productization/', 200, 'Show productization', 'firefox'],
+    ['showrepos/?locale=en-GB', 200, 'Health Status for en-GB', 'General metrics'],
+    ['stats/', 200, 'Repository status overview', 'Status estimate'],
+    ['string/?entity=apps/sms/ssms.properties:home&repo=gaia', 200, 'All translations for this string', 'supported_locales'],
```

Comment:
typo: sms.properties.
It works because the title is displayed even if the data is not found. But I think it would make sense to test data that is available both locally and in Travis (https://github.com/mozfr/transvision/blob/master/tests/testfiles/TMX/ ). We could check for instance that for the string bookmarkResultLabel in mozilla-central, the translation Marque-page is displayed in the page, like that:

['string/?entity=browser/chrome/browser/places/places.properties:bookmarkResultLabel&repo=central', 200, 'All translations for this string', 'supported_locales', 'Marque-page'],

This should work in Travais as well.